README.md

```diff
@@ -1,2 +1,23 @@
-A leaflet plugin to abstract away some of details of adding data layers so that users just need to describe their data with a json object rather than adding each layer with redundant imperative code. This project is definitely a work-in-progress and open to contributions. 
+# leaflet-declarative-layers
```

Comment:
Hey, @justindra, do you think the readme is accessible enough for devs of all levels? They may not know what declarative and imperative refer to, but do you think they get the general idea that the plugin aims to make adding map layers easier by using a json object to describe the data and they dont need to worry (as much) about implementation details. 

Maybe ``` A Leaflet plugin to make adding map layers easier. ... ``` Should I drop the 'declarative'/'imperative' blabber?

Reply:
Yea, I think the whole declarative/imperative might be a bit much if you are aiming it for all levels. 

I think some examples here would be great. Especially to compare the steps required to set something up with and without this plugin.

Reply:
Good idea, some basic code snippets in the readme are a good idea.